src/main/java/duke/ToDos.java

```diff
@@ -0,0 +1,25 @@
+package duke;
+
+import duke.Task;
+/**
+ * This class is implementation of Task which contains description.
+ */
+public class ToDos extends Task {
```

Comment:
I feel like you should name your class as `Todo` instead of `Todos` since this is technically not a collection of Todos. Also considering that you named your parent class `Task` instead of `Tasks` and `Deadline` instead of `Deadlines` I feel that you should stick to the singular naming convention.